pos_order_return/models/pos_order.py

```diff
@@ -51,9 +51,9 @@ def _prepare_invoice(self):
         if not self.returned_order_id.invoice_id:
             return res
         res.update({
-            'origin': self.returned_order_id.invoice_id.number,
+            'origin': self.returned_order_id.invoice_id.document_number,
```

Comment:
Hi @emavarela. what is this field ? 
Travis is failing on that line : https://travis-ci.com/github/OCA/pos/jobs/498674882#L1886

regards.